kimchi/src/prover.rs

```diff
@@ -650,123 +688,191 @@ where
                 })
         };
 
-        // evaluate the polynomials
-        let chunked_evals_zeta = ProofEvaluations::<Vec<Fr<G>>> {
-            s: array_init(|i| index.cs.sigmam[0..PERMUTS - 1][i].eval(zeta, index.max_poly_size)),
-            w: array_init(|i| witness_poly[i].eval(zeta, index.max_poly_size)),
-            z: z_poly.eval(zeta, index.max_poly_size),
-            lookup: lookup_evals(zeta),
-            generic_selector: index.cs.genericm.eval(zeta, index.max_poly_size),
-            poseidon_selector: index.cs.psm.eval(zeta, index.max_poly_size),
-        };
-        let chunked_evals_zeta_omega = ProofEvaluations::<Vec<Fr<G>>> {
-            s: array_init(|i| {
-                index.cs.sigmam[0..PERMUTS - 1][i].eval(zeta_omega, index.max_poly_size)
-            }),
-            w: array_init(|i| witness_poly[i].eval(zeta_omega, index.max_poly_size)),
-            z: z_poly.eval(zeta_omega, index.max_poly_size),
-            lookup: lookup_evals(zeta_omega),
-            generic_selector: index.cs.genericm.eval(zeta_omega, index.max_poly_size),
-            poseidon_selector: index.cs.psm.eval(zeta_omega, index.max_poly_size),
+        //~ 29. Chunk evaluate the following polynomials at both $\zeta$ and $\zeta \omega$:
```

Comment:
do we ever give the intuition that $\zeta$ refers to a row $i$ and $\zeta\omega$ to row $i+1$? if so, where?

Reply:
We don't. I'd be a great addition though, not sure where we could add that in the spec :o if you want to give it a shot be my guest!